src/main/scala/com/scalableminds/fossildb/db/RocksDBStore.scala

```diff
@@ -28,9 +29,21 @@ class RocksDBManager(dataDir: Path, columnFamilies: List[String]) extends LazyLo
       new ColumnFamilyDescriptor(columnFamily, columnOptions)
     }
     val columnFamilyHandles = new util.ArrayList[ColumnFamilyHandle]
-    val options = new DBOptions()
+    var options = new DBOptions()
       .setCreateIfMissing(true)
       .setCreateMissingColumnFamilies(true)
+    var cfListRef: mutable.Buffer[ColumnFamilyDescriptor] = mutable.Buffer()
```

Comment:
@fm3: this is not used currently, right?

Reply:
correct, but it has to exist and to be passed (by reference) to the loadOptionsFromFile